src/main/java/se/sundsvall/chat/api/model/ConversationRequest.java

```diff
@@ -0,0 +1,102 @@
+package se.sundsvall.chat.api.model;
+
+import io.swagger.v3.oas.annotations.media.ArraySchema;
+import io.swagger.v3.oas.annotations.media.Schema;
+import java.util.List;
+import java.util.Objects;
+
+@Schema(description = "Represents a conversation containing participants, messages, and metadata.")
+public class ConversationRequest {
```

Comment:
Do we need a separate request class for conversation? We should be able to mark attributes as readOnly and reuse model right?